packages/terra-demographics-banner/package.json

```diff
@@ -21,7 +21,8 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
-    "terra-toolkit": "^0.4.0"
+    "terra-toolkit": "^0.4.0",
+    "react-intl": "^2.3.0"
```

Comment:
Yeap. It's for the jest testing. To test, it should throw expected error message, if there is no intl in context. [code link](https://github.com/cerner/terra-core/pull/431/files#diff-b906926b8eeeef355e489c2e716c19b0R3)